libmscore/score.cpp

```diff
@@ -4417,6 +4430,40 @@ QString Score::nextRehearsalMarkText(RehearsalMark* previous, RehearsalMark* cur
     }
 }
 
+//---------------------------------------------------------
+//   composer
+//---------------------------------------------------------
+
+QString Score::composer() const
+{
+    for (auto x : _measures.first()->el()) {
+        if (x && x->isText()) {
+            Text* t = toText(x);
+            if (t->tid() == Tid::COMPOSER) {
```

Comment:
this is hacky, because users can absolutely style other bits of text with the 'composer' `Tid`, but IIRC there probably isn't a better way to do this.

This may cause problems if a composer's name is, for whatever reason, displayed on other pages of a score.

Reply:
Hm.... I hadn't thought of that but I believe that I will think of something to deal with it.